app/models/user.rb

```diff
@@ -82,7 +82,11 @@ def client_admin?
   end
 
   def admin?
-    self.has_role?('admin')
+    has_role?('admin')
+  end
+
+  def not_admin?
+    !admin?
   end
```

Comment:
@afeld yeah, basically I was trying to get away from rubocop blowing up on me because of an `if !` (rubocop wants that to be an `unless`)...don't think all booleans should have two methods